pysparkle/pysparkle/data_quality/config.py

```diff
@@ -0,0 +1,29 @@
+from pydantic import BaseModel, Field
+
+
+class Expectation(BaseModel):
+    expectation_type: str = Field(description="The name of the GX expectation class to use.")
+    expectation_kwargs: dict = Field(
+        description="The keyword arguments to pass to the expectation class."
+    )
+
+
+class ValidationConfig(BaseModel):
+    column_name: str
+    expectations: list[Expectation]
+
+
+class DatasourceConfig(BaseModel):
+    datasource_name: str = Field(description="Data asset name (e.g. table or file name).")
+    datasource_type: str = Field(description="Source system type (e.g. table, parquet, csv).")
+    data_location: str = Field(
+        description="Path to the given data asset or a fully qualified table name."
```

Comment:
As above, perhaps clarify in here for table it needs to be a hive metastore table location, for other types it needs to file path.

Reply:
Done